packit/config/aliases.py

```diff
@@ -20,6 +20,10 @@
     "fedora-latest-stable": ["fedora-39"],
     "fedora-branched": ["fedora-38", "fedora-39"],
     "epel-all": ["epel-7", "epel-8", "epel-9"],
+    "opensuse-leap-all": (
+        leap_all := ["opensuse-leap-15.5", "opensuse-leap-15.4", "opensuse-leap-15.3"]
+    ),
+    "opensuse-all": ["opensuse-tumbleweed", *leap_all],
```

Comment:
Thanks for covering this as well!

Not sure if you know but this is only a backup if we can't get the correct info from some authoritative source ad-hock  -- (As you can see in `get_aliases()` below...)

Do you think you can do something similar as well? We've slightly touched this with @dcermak and I can't remember what source/API he has suggested...

This can definitely be done in a separate PR.

Reply:
Also, the current Packit code has been moved into https://github.com/rpm-software-management/fedora-distro-aliases and Packit hasn't moved to it yet but it would be nice to make this contribution there if possible. I'll discuss renaming the project into `distro-aliases` to be distro-agnostic.

Reply:
I discussed this yesterday with Copr team maintaining the `fedora-distro-aliases` and they see some possible issues with including this in this project and suggest creating a similar one for SUSE instead to allow people to choose what they want. (And maybe have one joined on top of this...) But if there are no extra dependencies, they are ok-ish with having it there.

If we don't want to spend much time on this, we can probably leave it in Packit for now...

Reply:
We can leave this for now. I still need to implement support for obs in packit-service

Reply:
@dcermak  what do you think?